### ISSUE TITLE
Why does result in Table IV is higher than main table?

### ISSUE THREAD

**Issue by Agito555** [PRBonn/SfmOcc#2]
While reading your paper, I noticed that the mIoU reported in the main results table (Table I) is 17.7, whereas Table IV reports a higher mIoU of 20.5. Could you clarify why the result in Table IV is higher than that in the main table?

**Comment by jbehley** [comment]
Note that we performed all our ablations on a subset of the training data (60 scenes) not used for training as noted in Sec. IV-F. Therefore, the performance of the final design choices based on these ablations are only evaluated on the test set.

Hope that clarifies your question.

**Comment by Agito555** [comment]
Thanks for replying.
It's uncommon to evaluate models on the training set. Could you please clarify why the ablation experiments were performed on the training set rather than the validation set?

**Comment by jbehley** [comment]
As the nuScenes validation set is used as test set in Occ3D-nuScenes, we decided to take some part of the training set as our validation set to perform ablations and hyper parameter choices, but also early stopping while training. 

The 60 scenes from the training set are **not** used for training and therefore we have a clean separation between test, train, and validation. 

Using this setup, we avoid over-fitting or making design decisions on the test set (i.e., the nuScenes validation set), which is common practice in machine learning.

Hope that clarifies our reasoning.

**Comment by Agito555** [comment]
Thanks for your answering, I will close the issue.
